instance/models/instance.py

```diff
@@ -89,6 +92,18 @@ def save(self, *args, **kwargs):
             'instance_id': self.pk,
         })
 
+    @classmethod
+    def can_manage(cls, user):
+        """
+        Returns true if the user is allowed to manage all instances.
+
+        User must:
+        * be a staff member, or
+        * have the "manage_all" permission
+        """
+        permission = '{}.{}'.format(cls._meta.app_label, "manage_all")
+        return user.is_staff or user.has_perm(permission)
```

Comment:
@bradenmacdonald @smarnach 

Currently, only Braden, Jacek, and Xavier are superusers in PRD, and the rest of us are just staff users.

As part of the deployment, I can add the new permission to the existing is_staff users, but then new people will have to remember to do that too.

I think it's simpler to just keep the `is_staff` check, and use this permission without `is_staff` for users that we want to give broad privileges too without giving them full access to /admin.